common/src/main/java/org/opensearch/ml/common/MLTaskType.java

```diff
@@ -8,7 +8,6 @@
 public enum MLTaskType {
     TRAINING,
     PREDICTION,
-    BATCH_PREDICTION,
```

Comment:
Changing the enum order will introduce bwc issue. We should only add enum object to the very last one

Reply:
Saw it from a previous wrong order. So this is actually a fix. Ignore previous comment